scripts/validate-mdx-links.ts

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node --experimental-strip-types --no-warnings=ExperimentalWarning
```

Comment:
I'm never writing a Bash script again.
Would prefer `bun`, but `node --experimental-strip-types` is pretty great.